.github/workflows/docs.yaml

```diff
@@ -45,12 +43,9 @@ jobs:
       - name: env setup
         run: |
           python -m pip install --upgrade pip
-          python -m pip install nbsite==0.8.0rc2
-          python -m pip install -e .[doc]
+          python -m pip install hatch
```

Comment:
Personal preference but I would use `pipx` to install `hatch`.

Reply:
I would use `pipx` mostly to isolate `hatch`. 

And it is available in the CI without installing it. I use it [here](https://github.com/Hoxbro/clean_notebook/blob/main/.github/workflows/test.yaml#L32)

Reply:
Ah interesting I didn't know it was available on the runners. But yeah as you said, that's personal preference at this stage, I think it's fine installing `hatch` with pip instead of with `pipx`. I'll change my mind when this breaks!